packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -3781,7 +3847,7 @@ function writeAssets(assets: { filePath: string; content: string }[]) {
 function mockUploadWorkerRequest(
   options: {
     available_on_subdomain?: boolean;
-    expectedEntry?: string;
+    expectedEntry?: string | RegExp;
```

Comment:
`expectedEntry` which is only passed into `toMatch` which has a signature of `string | RegExp`

Reply:
we don't need to pass a regex, it'll already match against substrings